product_docs/docs/pem/8/pem_online_help/08_toc_pem_developer_tools/04_editgrid/index.mdx

```diff
@@ -80,6 +80,45 @@ Notes:
 -   `Data size:` For performance reasons, the viewer will render no more than 100000 geometries, totaling up to 20MB.
 -   `Internet access:` An internet connection is required for the Geometry Viewer to function correctly.
 
+**JSON Data Editor**
+
+A built in json editor is provided for *JSON/JSONB Data*.Double clicking on json/jsonb data type cell in data grid will open JSON Editor.Editor provides different mode to view and edit json data.
+
+<img src="../../images/json_editor_modes.png" class="align-center" alt="Json Editor" />
```

Comment:
The syntax for images need to be replaced with ! [Alternate_text](<path_to_image_file>/<image_filename.png>)